test/validators.js

```diff
@@ -4291,6 +4294,8 @@ describe('Validators', () => {
         'rgba(3,3,3%,.3)',
         'rgb(101%,101%,101%)',
         'rgba(3%,3%,101%,0.3)',
+        'r         g    b(   0,         251,       222     )',
+        'r         g    ba(   0,         251,       222     )',
       ],
```

Comment:
I think another testcase like so could be useful, just to be on the safe side :)

```suggestion
        'r         g    ba(   0,         251,       222     )',
        rg ba(0, 251, 22, 0.5)',
      ],
```